openy.profile

```diff
@@ -168,6 +168,10 @@ function openy_demo_content_configs_map($key = NULL) {
     ],
   ];
 
+  \Drupal::service('module_installer')->install([
```

Comment:
This shouldn't be here. Eventually our credentials will be setup for all installations, but it shouldn't be so. Even when user don't need demo content.

1. Add new set of chekboxes:

$form['settings'] = [
   '#type' => 'checkboxes',
  '#title' => $this->t('Choose settings to import'),
  '#default_value' => array_keys($options),
  '#options' => $settings:options,
];

2. Add options `Addthis settings` to this new section

3. Implement same condition in function `openy_import_content ` as here https://github.com/ymcatwincities/openy/blob/8.x-1.x/openy.profile#L187